Add 1.15.0 release notes and announcement

Monkeyanator opened the pull request:
Adds 1.15.0 release notes and announcement. Seems that no upgrade notes were added for 1.15.

istio-testing commented:
@Monkeyanator: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio.io | d9de818130709d79531e4a044f2735c5dc949128 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/11759/lint_istio.io/1563282764118102016) | true | `/test lint`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@Monkeyanator: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.profile_default_istio.io | 9de9fd45c42d55159039d304734a83c0e3da55bc | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/11759/doc.test.profile_default_istio.io/1563308699269730304) | true | `/test doc.test.profile_default`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

ericvn commented:
/test doc.test.profile_default

frankbu commented on content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md:
```diff
@@ -0,0 +1,142 @@
+
+- **Updated** the Kiali addon to the latest version (v1.55).
+
+- **Added** support for watching local secret resource updates for external istiod.
+  ([Issue #31946](https://github.com/istio/istio/issues/31946))
```
```suggestion
- **Added** support for watching local secret resource updates for external istiod.
  ([Issue #31946](https://github.com/istio/istio/issues/31946))

- **Improved** [external control plane setup instructions](/docs/setup/install/external-controlplane/), including tips for simpler control plane ingress setup, making it easier to experiment with the external control plane deployment model in a test environment.
```

Monkeyanator replied:
Done.

ericvn commented:
We are still missing the page based on the Google Doc.

ericvn commented on content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md:
```diff
@@ -0,0 +1,142 @@
+- **Added** support for parsing list type of files in `istioctl analyze`.
+  ([Issue #39982](https://github.com/istio/istio/issues/39982))
+
+- **Added** description to `istioctl admin log`
```
Add a period to the send of the sentence.

Monkeyanator replied:
Done.

ericvn commented on content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md:
```diff
@@ -0,0 +1,142 @@
+
+- **Updated** istiod to allow unknown flags for backward-compatibility. If an unknown flag is passed, no warning or error will be logged.
+
+- **Added** add a validation warning when protocol is unset and address is also unset.
```
~~add~~ don't need double add.

Monkeyanator replied:
Done.

ericvn commented:
One thing I noticed is the items aren’t sorted by Action under each topic. Normally its the Added, then Deprecated continuing alphabetically up to Upgraded. See https://istio.io/latest/news/releases/1.14.x/announcing-1.14/change-notes/ for an example. You can certainly order in other ways, like what's maybe a higher priority, and that may have been done already.

ericvn commented:
> Seems that no upgrade notes were added for 1.15.

I also looked and didn't see any upgradeNotes

Monkeyanator commented:
> I also looked and didn't see any upgradeNotes. This may mean we only have something like:

Added blank upgrade notes section with that text.

>  You can certainly order in other ways, like what's maybe a higher priority, and that may have been done already.

Yeah I'd shuffled a bit based on importance. Reshuffled so that the verbs are together.

> We are still missing the page based on the Google Doc.

Added

ericvn commented on content/en/docs/releases/supported-releases/index.md:
```diff
@@ -51,18 +51,19 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
+| Version         | Currently Supported  | Release Date      | End of Life              | Supported Kubernetes Versions | Tested, but not supported    |
+|-----------------|----------------------|-------------------|--------------------------|-------------------------------|------------------------------|
+| master          | No, development only |                   |                          |                               |                              |
+| 1.15            | Yes                  | August 31, 2022   | ~March 2023 (Expected)   | 1.22, 1.23, 1.24, 1.25        | 1.16, 1.17, 1.18, 1.19, 1.20 |
```
Missing 1.21 in last column .

Monkeyanator replied:
Added.

ericvn commented on content/en/news/releases/1.15.x/_index.md:
```diff
@@ -0,0 +1,46 @@
+{{< relnote >}}
+
+This is the third Istio release of 2022. We would like to thank the entire Istio community
+for helping to get Istio 1.15.0 published. Special thanks are due to the release managers Sam Naser and Aryan Gupta from Google, Ziyang Xiao from Intel and Daniel Hawton from solo.io. As always, our gratitude goes to Test & Release WG lead Eric Van Norman (IBM) for his help and guidance.
```
I expect solo.io here should be Solo.io or just Solo. I'm not sure how they differentiate whether.io should be added. I do see on their website, `At Solo.io, we have helped thousands of organizations` which means that they typically capitalize it unless it's the URL...

ericvn replied:
@dhawton Thoughts on Solo vs Solo.io?

dhawton replied:
Solo.io is preferred.

ericvn commented on content/en/news/releases/1.15.x/_index.md:
```diff
@@ -0,0 +1,46 @@
+layout: release-grid
+decoration: dot
+---
+
```
The first 8 lines need to stay here, but the rest of the doc goes into content/en/news/releases/1.15.x/announcing-1.15/_index.md (so one level deeper in the tree).

You can copy the first few lines from 1.14 for example. I don't think you need aliases at this point.

Monkeyanator replied:
Fixed file structure and moved one level deeper.

ericvn commented on content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md:
```diff
@@ -0,0 +1,144 @@
+release: 1.15.0
+weight: 10
+aliases:
+    - /news/announcing-1.15.0
```
You don't need the alias.

Monkeyanator replied:
Removed.

Monkeyanator commented on .spelling:
```diff
@@ -855,6 +860,7 @@ Snell-Feikema
 SNI
 SolarWinds
 Solo.io
+solo.io.
```
Removed line (swapped for `Solo.io` on the page)

istio-testing commented:
@Monkeyanator: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio.io | 112fa678fe2465baf96b7515f0123e765367cf7f | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/11759/lint_istio.io/1564829227906240512) | true | `/test lint`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

ericvn reviewed (approved):
Merge this in preparation for the branch cut.
